data/json/terrain.json

```diff
@@ -257,6 +269,7 @@
         "type" : "terrain",
         "id" : "t_ov_reb_cage",
         "name": "rebar roof cage",
+        "description": "A network of architecturally placed rebar in order to support a floor and roof, looks like it's missing the poured concrete.",
```

Comment:
Not sure what `architecturally placed` means

Recommend switching the comma to a period

Reply:
>Looks like it's missing the poured concrete

would be an incomplete sentence.

Architecturally placed means designed carefully essentially. Not just haphazardly assembled.

Perhaps `<addr>` architecturally sound `<addr>`